projects/plugins/jetpack/extensions/blocks/writing-prompts/editor.js

```diff
@@ -0,0 +1,27 @@
+import { createBlock } from '@wordpress/blocks';
+import { dispatch } from '@wordpress/data';
+import { waitForEditor } from '../../shared/wait-for-editor';
+
+async function insertTemplate( prompt, embedPrompt = false ) {
+	await waitForEditor();
+
+	const { insertBlocks } = dispatch( 'core/block-editor' );
+	const writingPromptBlocks = embedPrompt
```

Comment:
I tested with `?embed_prompt=1` but it didn't seem to embed the content properly
<img width="732" alt="Screen Shot 2022-10-25 at 11 15 26 am" src="https://user-images.githubusercontent.com/22446385/197659313-931e8874-6fb3-4219-af6c-dbe0857904cd.png">